docs/user-guide/getting-started.md

```diff
@@ -1,185 +1,202 @@
 ---
-    description: In this guide, we'll walk through the process of deploying Ambassador Edge Stack in Kubernetes for ingress routing.
+   description: In this guide, we'll walk through the process of deploying Ambassador Edge Stack in Kubernetes for ingress routing.
 ---
 # Quick Start Installation Guide
 
 In this guide, we'll walk you through installing and configuring the Ambassador Edge Stack in your Kubernetes cluster. Within a few minutes, your cluster will be routing HTTPS requests from the Internet to a backend service. You'll also have a sense of how the Ambassador Edge Stack is managed.
 
+Different options for installation include:
+
+* **[Quick Install (recommended!)](#quick-install)**
+* [Install via Minikube](#minikube-users)
+* [Install in CI](#install-in-ci)
+
+Or, you can [install manually](/user-guide/manual-install) if you want to
+customize your configuration.
+
 ## Before You Begin
 
 The Ambassador Edge Stack is designed to run in Kubernetes for production. The most essential requirements are:
 
 * Kubernetes 1.11 or later
 * The `kubectl` command-line tool
+* [Edge Control](/reference/edgectl-download)
+
+## Quick Install (Recommended!)
+
+The Ambassador Edge Stack is typically deployed to Kubernetes from the command line. If you don't have Kubernetes, you should use our [Docker](../../about/quickstart) image to deploy the Ambassador Edge Stack locally. Or, if you're a Minikube user, [check out these directions](#minikube-users).
 
-## Install the Ambassador Edge Stack
+When you use Edge Control on your publicly
+accessible cluster, it will:
 
-The Ambassador Edge Stack is typically deployed to Kubernetes from the command line. If you don't have Kubernetes, you should use our [Docker](../../about/quickstart) image to deploy the Ambassador Edge Stack locally.
+1. Install the Ambassador Edge Stack
+2. Generate a domain name for you to access the Edge Policy Console and complete
+   advanced configuration
+3. Obtain a TLS certificate for that domain name
+4. Configure automatic TLS and HTTPS using that certificate
 
-1. In your terminal, run the following command:
+Install Edge Control for your operating system [here](/reference/edgectl-download). When complete, return to these instructions.
 
-    ```bash
-    kubectl apply -f https://www.getambassador.io/yaml/aes-crds.yaml && \
-    kubectl wait --for condition=established --timeout=90s crd -lproduct=aes && \
-    kubectl apply -f https://www.getambassador.io/yaml/aes.yaml && \
-    kubectl -n ambassador wait --for condition=available --timeout=90s deploy -lproduct=aes
-    ```
+**To install the Ambassador Edge Stack:**
 
-2. Determine the IP address of your cluster by running the following command:
+1. Run the following command: `edgectl install`
+2. Provide an email address so you can be notified before your domain and
```

Comment:
Better would be "During the installation process, you will be asked for your email address because the ACME provider of TLS certificates (Let's Encrypt) requires your email address.